### PR TITLE
Clean, remove and adapt proxy tests to be ran on thor

### PR DESCRIPTION
**Pull request by MakisChristou**

### CODE REVIEW

File: test/proxy/Clones.test.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ const { expectEvent } = require('@openzeppelin/test-helpers');
 const { expect } = require('chai');
 const { computeCreate2Address } = require('../helpers/create');
 const { expectRevertCustomError } = require('../helpers/customError');
```
Comment (paologalligit) [inline test/proxy/Clones.test.js]: This and all the occurrences in the other files can be removed

File: test/proxy/ERC1967/ERC1967Utils.test.js [hunk 2]
```diff
@@ -21,152 +21,152 @@ contract('ERC1967Utils', function (accounts) {
...
-          await expectRevertCustomError(
-            this.utils.$upgradeBeaconToAndCall(newBeacon.address, '0x'),
-            'BeaconProxyBeaconSlotAddress',
-            [newBeacon.address],
-          );
-        });
-      });
-    });
-  });
+  // describe('IMPLEMENTATION_SLOT', function () {
```
Comment (paologalligit) [inline test/proxy/ERC1967/ERC1967Utils.test.js]: Are all these lines to be removed or uncommented?

File: test/proxy/transparent/ProxyAdmin.test.js [hunk 3]
```diff
@@ -22,10 +23,16 @@ contract('ProxyAdmin', function (accounts) {
...
     const proxy = await TransparentUpgradeableProxy.new(this.implementationV1.address, proxyAdminOwner, initializeData);
 
-    const proxyNonce = await web3.eth.getTransactionCount(proxy.address);
-    const proxyAdminAddress = computeCreateAddress(proxy.address, proxyNonce - 1); // Nonce already used
-    this.proxyAdmin = await ProxyAdmin.at(proxyAdminAddress);
+    let txHash = proxy.transactionHash
+    let txReceipt = await web3.eth.getTransactionReceipt(txHash)
+    let proxyAdminAddress = txReceipt.logs[2].address
 
+    // const proxyNonce = await web3.eth.getTransactionCount(proxy.address);
```
Comment (paologalligit) [inline test/proxy/transparent/ProxyAdmin.test.js]: If we don't need these commented lines, we can remove them

File: test/proxy/transparent/ProxyAdmin.test.js [hunk 4]
```diff
@@ -40,38 +47,38 @@ contract('ProxyAdmin', function (accounts) {
...
         );
       });
     });
 
     context('with authorized account', function () {
-      it('upgrades implementation', async function () {
-        await this.proxyAdmin.upgradeAndCall(this.proxy.address, this.implementationV2.address, '0x', {
-          from: proxyAdminOwner,
-        });
+      // it('upgrades implementation', async function () {
```
Comment (paologalligit) [inline test/proxy/transparent/ProxyAdmin.test.js]: Same here, do we need to keep them or remove them?

File: test/proxy/transparent/ProxyAdmin.test.js [hunk 5]
```diff
@@ -89,14 +96,14 @@ contract('ProxyAdmin', function (accounts) {
...
       context('with valid callData', function () {
-        it('upgrades implementation', async function () {
-          const callData = new ImplV1('').contract.methods.initializeNonPayableWithValue(1337).encodeABI();
-          await this.proxyAdmin.upgradeAndCall(this.proxy.address, this.implementationV2.address, callData, {
-            from: proxyAdminOwner,
-          });
-          const implementationAddress = await getAddressInSlot(this.proxy, ImplementationSlot);
-          expect(implementationAddress).to.be.equal(this.implementationV2.address);
-        });
+        // it('upgrades implementation', async function () {
```
Comment (paologalligit) [inline test/proxy/transparent/ProxyAdmin.test.js]: Same here

File: test/proxy/transparent/TransparentUpgradeableProxy.test.js [hunk 6]
```diff
@@ -20,5 +20,5 @@ contract('TransparentUpgradeableProxy', function (accounts) {
   };
 
   shouldBehaveLikeProxy(createProxy, otherAccounts);
-  shouldBehaveLikeTransparentUpgradeableProxy(createProxy, owner, otherAccounts);
+  // shouldBehaveLikeTransparentUpgradeableProxy(createProxy, owner, otherAccounts);
```
Comment (paologalligit) [inline test/proxy/transparent/TransparentUpgradeableProxy.test.js]: Same here

File: test/proxy/utils/UUPSUpgradeable.test.js [hunk 7]
```diff
@@ -29,43 +29,38 @@ contract('UUPSUpgradeable', function () {
     expect(await this.instance.UPGRADE_INTERFACE_VERSION()).to.equal('5.0.0');
   });
 
-  it('upgrade to upgradeable implementation', async function () {
-    const { receipt } = await this.instance.upgradeToAndCall(this.implUpgradeOk.address, '0x');
-    expect(receipt.logs.filter(({ event }) => event === 'Upgraded').length).to.be.equal(1);
-    expectEvent(receipt, 'Upgraded', { implementation: this.implUpgradeOk.address });
-    expect(await getAddressInSlot(this.instance, ImplementationSlot)).to.be.equal(this.implUpgradeOk.address);
-  });
+  // it('upgrade to upgradeable implementation', async function () {
```
Comment (paologalligit) [inline test/proxy/utils/UUPSUpgradeable.test.js:32]: Same here, and all the remaining lines in this file

**Review by paologalligit — CHANGES_REQUESTED** [review]

File: test/proxy/ERC1967/ERC1967Utils.test.js (hunk 2 above)
Comment (MakisChristou) [reply test/proxy/ERC1967/ERC1967Utils.test.js]: Those are to be removed.

**Review by paologalligit — APPROVED** [review]

**Pull request merged**